tests/data/samplesheets/samplesheet-loci-mismatch.csv

```diff
@@ -0,0 +1,4 @@
+sample,mlst_alleles,metadata_1,metadata_2,metadata_3,metadata_4,metadata_5,metadata_6,metadata_7,metadata_8
+sample1,https://raw.githubusercontent.com/phac-nml/gasclustering/dev/tests/data/reports/sample1.mlst.json,,,,,,,,
+sample2,https://raw.githubusercontent.com/phac-nml/gasclustering/dev/tests/data/reports/sample2.mlst.json,,,,,,,,
+sample_mismatch,/home/eric/projects/gasclustering/tests/data/reports/sample_mismatch.mlst.json,,,,,,,,
```

Comment:
Friendly reminder to update the path when you merge to dev 😄

Reply:
Removed this file in 2e0df69c8166ebf6d0e518630d40ebe36f7f6266. I think I accidentally created two samplesheets for the new test but only used one of them.